AirCasting/Services/DownloadMeasurmentsService.swift

```diff
@@ -83,4 +89,22 @@ final class DownloadMeasurementsService: MeasurementUpdatingService {
             }
         })
     }
+    
+    private func calculateLastSync(sessionEndTime: Date?, lastMeasurementTime: Date?) -> Date {
+        let measurementTimeframe: Double = 24 * 60 * 60 // 24 hours in seconds
+        
+        guard let sessionEndTime = sessionEndTime else { return Date() }
+        let sessionEndTimeSeconds = sessionEndTime.timeIntervalSince1970
+
+        let last24hours = Date(timeIntervalSince1970: (sessionEndTimeSeconds - measurementTimeframe))
+        
+        guard let lastMeasurementTime = lastMeasurementTime else { return last24hours }
+        let lastMeasurementSeconds = lastMeasurementTime.timeIntervalSince1970
+        
+        if (sessionEndTimeSeconds - lastMeasurementSeconds) <  measurementTimeframe {
```

Comment:
Please remove one of two spaces  `lastMeasurementSeconds) <  measurementTimeframe {`